dbt/adapters/databricks/impl.py

```diff
@@ -131,13 +131,13 @@ def list_relations_without_caching(  # type: ignore[override]
                     f'Invalid value from "show table extended ...", '
                     f"got {len(row)} values, expected 4"
                 )
-            _, name, _, information = row
+            _schema, name, _, information = row
```

Comment:
Can we comment here on why we need the schema returned from the row?

Reply:
Sure, left a comment.
Actually we might have a better way to handle this, but let me just revert to avoid the current regression.

Reply:
Sounds good!